src/main/docker/startup/startup.sh

```diff
@@ -16,16 +16,17 @@
 #
 
 # Create a convenience function for logging
+processId = $BASHPID;
```

Comment:
What is the idea of this line?

Reply:
Without this, if we use $BASHPID in echo, each time it generates new pid for the startup.sh. All log method is called from the startup.sh. But it generates new PID for each log method call. The pid should be same for all log method call inside the startup.sh file.

Please see the build for the issue:
https://jenkins.hpeswlab.net/job/SEPG/job/caf/job/caf~worker-elastic~SCMOD-7472~CI/6/consoleFull
After Fix:
https://jenkins.hpeswlab.net/job/SEPG/job/caf/job/caf~worker-elastic~SCMOD-7472~CI/7/consoleFull